doc/changelog.md

```diff
@@ -153,71 +283,89 @@
+  [(#705)](https://github.com/PennyLaneAI/catalyst/pull/705)
+  [(#710)](https://github.com/PennyLaneAI/catalyst/pull/710)
 
-  There are no changes to the syntax of inactive callbacks, including
-  `debug.print`. See the following example:
+  The decorator `catalyst.disable_autograph` allows one to disable Autograph
+  from auto-converting specific external functions when called within a qjit-compiled
+  function with `autograph=True`:
 
   ```python
-  @qml.qjit
-  @grad
-  def identity(x: float):
-    debug.print(x)
-    return x
+  def approximate_e(n):
+      num = 1.
+      fac = 1.
+      for i in range(1, n + 1):
+          fac *= i
+          num += 1. / fac
+      return num
+  
+  @qml.qjit(autograph=True)
+  def g(x: float, N: int):
+  
+      for i in range(N):
+          x = x + catalyst.disable_autograph(approximate_e)(10) / x ** i
+  
+      return x
   ```
 
-  Parameters to `pure_callback`s are active variables. This means the
-  partial derivative of `pure_callback`s needs to be computed.
-  Since callbacks are opaque to the compiler, the user needs to register
-  custom gradients with Enzyme.
+  ```pycon
+  >>> g(0.1, 10)
+  array(4.02997319)
+  ```
 
-  In order to differentiate `pure_callback`s please use the following syntax:
+  Note that for Autograph to be disabled, the decorated function must be
+  defined **outside** the qjit-compiled function. If it is defined within
+  the qjit-compiled function, it will continue to be converted with Autograph.
 
-  ```python
-  @pure_callback
-  def identity(x) -> float:
-    return x
+  In addition, Autograph can also be disabled for all externally defined functions
+  within a qjit-compiled function via the context manager syntax:
 
-  @identity.fwd
-  def fwd(x):
-    return identity(x), 1.0
+  ```python
+  @qml.qjit(autograph=True)
+  def g(x: float, N: int):
+  
+      for i in range(N):
+          with catalyst.disable_autograph:
+            x = x + approximate_e(10) / x ** i
+  
+      return x
+  ```
 
-  @identity.bwd
-  def bwd(res, cot):
-    return cot * res
+* Support for including a list of (sub)modules to be whitelisted for autograph conversion.
+  [(#725)](https://github.com/PennyLaneAI/catalyst/pull/725)
 
-  @qml.qjit
-  @grad
-  def wrapper(x):
-    return scale * identity(x)
-  ```
+  Although library code is not meant to be targeted by Autograph conversion,
+  it sometimes make sense to enable it for specific submodules that might
+  benefit from such conversion:
 
-  The forward pass must always return a tuple where the first element of the tuple
-  is the result of the function being differentiated and the second element of the tuple
-  are the residuals that may be used in the reverse pass. There may be multiple residuals
-  and also no residuals. In the case of no residuals, please return `None` like so:
+  ```py
+  @qjit(autograph=True, autograph_include=["excluded_module.submodule"])
+  def f(x):
+    return excluded_module.submodule.func(x)
```

Comment:
I've added the following sentence: 

>   For example, this might be useful if importing functionality from PennyLane (such as
  a transform or decomposition), and would like to have Autograph capture and convert
  associated control flow.